client/packages/common/src/ui/components/inputs/DatePickers/DatePickerInput/DatePickerInput.tsx

```diff
@@ -33,6 +37,8 @@ export const DatePickerInput: FC<DatePickerInputProps> = ({
       width={width}
       error={error}
       label={label}
+      minDate={minDate}
+      maxDate={maxDate}
       textFieldProps={textFieldProps}
```

Comment:
I'm starting to think that this "DatePickerInput", which is just a wrapper over "BaseDatePickerInput" is a bit unnecessary, so we might want to think about removing it at some point and just using the Base directly (renaming it)

Reply:
yep